src/measure_theory/interval_integral.lean

```diff
@@ -647,8 +694,235 @@ begin
     simp [h] }
 end
 
+lemma congr_ae' {f g : α → E} (h : ∀ᵐ x ∂μ, x ∈ Ioc a b → f x = g x)
+  (h' : ∀ᵐ x ∂μ, x ∈ Ioc b a → f x = g x) :
+  ∫ (x : α) in a..b, f x ∂μ = ∫ (x : α) in a..b, g x ∂μ :=
+by simp only [interval_integral, set_integral_congr_ae (measurable_set_Ioc) h,
+              set_integral_congr_ae (measurable_set_Ioc) h']
+
+lemma congr_ae {f g : α → E} (h : ∀ᵐ x ∂μ, x ∈ Ι a b → f x = g x) :
```

Comment:
Judging by the other lemmas in this file, these should probably be `integral_congr_ae'` and `integral_congr_ae`.